src/app/components/core/north/add-task-wizard/add-task-wizard.component.html

```diff
@@ -149,7 +149,7 @@
           <div class="column">
             <div class="config-div">
               <div class="field is-horizontal">
-                <div class="field-label has-text-left">
+                <div class="field-label has-text-centered enabled-field">
```

Comment:
step1 labels are  right,
step2 labels are  left,
step2 labels are  centered?!,

Reply:
Yes, `has-text-centered` fixed the alignment issue in step3.